src/modules/SuSEFirewallCMDLine.rb

```diff
@@ -1534,30 +1547,53 @@ def FWCMDMasqRedirect(options)
     # @return [Boolean] whether write call is needed
     def FWCMDMasquerade(options)
       options = deep_copy(options)
+      zone = nil
+      if firewalld?
+        if options["zone"] != nil
```

Comment:
still `!= nil` is not needed, nil is false value, so it can be simple `if options["zone"]`